intermediate/foundations.md

```diff
@@ -36,6 +49,14 @@ about what's going on and less unmotivated symbol-pushing than in E/F/T.  The fl
 is, the constructions may or may not be epsilon less precise.  I'm not a logician; if you are,
 write some reviews so I can replace these lousy ones!
 
+#### Chang & Kiesler, *Model Theory*
```

Comment:
Spelling -- I believe this is 'Keisler'?

Reply:
Oops, yes it is Keisler.